Week6.js

```diff
@@ -0,0 +1,121 @@
+
+// Q1
+//console.log('Hello world!');
+//console.log('!مرحبا بالعالم');
+//console.log('Saluton mondo');
+//console.log('Hej världen!');
+//console.log('Selam Dünya!');
+//console.log('ہیلو ورلڈ!');
+//console.log('Aloha kākou honua!');
+
+//Q2
+//console.log('Im awesome');
+
+
+/*
+//Q3
+let age;
+console.log ('You still Young, your age is:');
```

Comment:
:smile:
the instructions were asking you what you think the value of `age` is on line 17 before you give it a value.